requirements.txt

```diff
@@ -1,4 +1,4 @@
-sphinx==3.5.4
+sphinx
```

Comment:
Yeah, the problem comes with older sphinx versions and their dependencies to other packages. With the fixed version v3.5.4 e.g., I get the following error after make html

...
ImportError: cannot import name 'environmentfilter' from 'jinja2' 

Using the newest sphinx version helps and no import errors appear.

I remember that we needed to fix the version (some time ago) because the at-this-time newest sphinx version had some small bugs.

PS: sorry for double posting ...